docs/components/topics.rst

```diff
@@ -0,0 +1,73 @@
+
+Permissions
+------------------------------------
+
+* Users with the Site Manager or Editor role can add and edit topic terms.
+* Users with the Administrator role can add new icons.
+
+Adding a new topic term
+------------------------------------
+
+From the Administation menu, navigate to ``Site Configuration > Taxonomy > Topics > Add term``
+
+:Name: Enter the term for your new topic.
+:Description: This field is not currently displayed publicly.
+:Icon Type: DKAN Topics comes with a default set of font icons that can be used with your terms, you can upload your own font icons if desired. See Adding new icons. Or you may select to use image icons, when you toggle the image option, an image upload input field will appear.
+:Icon: If using font icons, select the icon you want to associate with your topic term.
+:Icon Color: Icons will display the same color as text on datasets unless a specific color is selected here.
+
+Editing topic terms
+------------------------------------
+
+1. From the Administation menu, navigate to ``Site Configuration > Taxonomy > Topics``
+2. You will see a list of current topic terms, click the 'edit' link under Operations that corresponds to the term you would like to edit. 
+3. Make changes and click "Save".
+
+Removing Topics from the main menu
+------------------------------------
+
+1. Navigate to ``Site Configuration > Menus``
+2. On the Menus screen, click "list links"
+3. Uncheck the box under "Enabled" for Topics
+4. Click "Save configuration"
+
+Adding new icons
+------------------------------------
+
+**3 Important notes:** 
+  * Only users with the Administrator role can add new icon fonts.
+  * The font can only be changed if there are **NO** default icon values in use. Only one icon font can be used at a time.
+  * The display of the icons in facets and the drop down menu will break.
+1. Navigate to ``Configuration > Content Authoring > Font Icon Select Options``
+2. Click "Upload New Library"
+3. Enter a title for your new font option
+4. Upload the four files for your font
+5. Click "Save"
+6. Navigate to ``Structure > Taxonomy > Topics > Manage Fields > Icon``
+7. Select your font from the font dropdown in the Icon field settings section.
+8. Click "Save settings"
+
+
+Known Issues
+------------------------------------
+
+* When enabling this module on an existing DKAN site, the default topics may be added to the top-level navigation menu. If this happens, simply use the taxonomy UI to move the new terms under the "Topics" term. New topics you add should then be added under the Topics term in the main menu.
+
+* The module adds a main menu link for "Topics". If you want a different word in place of "Topics", you will need to modify the term from the taxonomy UI. This will cause the DKAN Topics feature to be overridden and make future upgrades more difficult.
```

Comment:
@dafeder if the menu title is changed, there are no overrides, but the icons will not display in the menu. If you use string overrides to change 'Topics' to something else, the menu works and the icons display. However, the facet title and form label remain 'Topics' so I will just make these clarifications under known issues and remove the old stuff.